src/xenia/cpu/xex_module.cc

```diff
@@ -281,6 +281,34 @@ bool XexModule::Load(const std::string& name, const std::string& path,
     }
   }
 
+  // Setup memory protection.
+  // TODO: This introduces a load of constants into the JIT, and Xenia isn't
```

Comment:
Ah yeah - maybe I can help with some of that. I still want to rework that file to make it more readable/etc. Adding a proper LIR would be nice, too.